### ISSUE TITLE
Community Diligence Review of BDX Allocator

### ISSUE THREAD

**Issue by filecoin-watchdog** [filecoin-project/Allocator-Governance#20]
Review of BDX Allocations from @cryptoAmandaL
Allocator Application: https://github.com/filecoin-project/notary-governance/issues/1027

First example:
DataCap was given to:
https://github.com/cryptoAmandaL/BDE-Allocator/issues/3

Public Open Dataset - key compliance requirement: Retrievability

SPs listed
f01841134 China/
f02029743 Singapore/
f0420161 Canada/
f02239446 HK/
f01844118 US

SPs taking deals.
Provider | Location | Total Deals Sealed | Percentage | Unique Data | Duplicate Deals | Mean Spark Retrieval Success Rate 7d
f03027241 | New York City, New York, USSingleHop LLC | 300.00 TiB | 41.44% | 300.00 TiB | 0.00% | -
f02816050 | London, England, GBSpeedyPage Ltd | 424.00 TiB | 58.56% | 424.00 TiB | 0.00% | -

No SP matches, no retrievals, and 3PiBs already given to one client by allocator with no diligence.

**Comment by Kevin-FF-USA** [comment]
Hi @cryptoAmandaL

On the next Fil+ Allocator meeting we will be going over each refill application. Wanted to ensure you were tracking the review discussion taking place in https://github.com/filecoin-project/Allocator-Governance/issues/20.

If your schedule allows, recommend coming to the May 28th meeting to answer/discuss the issues raised in the recent distributions. This will allow you to faster address - or, the issue in Allocator Governance for ongoing written discussion.

Warmly,
-Kevin
https://calendar.google.com/calendar/embed?src=c_k1gkfoom17g0j8c6bam6uf43j0%40group.calendar.google.com&ctz=America%2FLos_Angeles

![image](https://github.com/filecoin-project/Allocator-Governance/assets/94004586/8d4a40a1-e993-483b-b487-42ddbcc9beb1)

**Comment by cryptoAmandaL** [comment]
Hello @filecoin-watchdog, 

thanks for your review.

a. There're a total of two clients applied to me before you opened this issue. It seems to be a problem with the [allocator.tech](https://allocator.tech/), I can't sign and allocate datacap to my first client via this website.
Since May 16th, I have spoken to Gov technician several times about this problem, but it has not been solved yet.

b. The client had contacted me that they had changed sps during the sealing process. I've already suggested to the client to increase the amount of sp to work with in the following process.
![image](https://github.com/filecoin-project/Allocator-Governance/assets/134273610/a8e48941-481c-4b28-8b67-3812ef89ab92)

From the actual cooperation of the client now, the client is already working with 5 different sps.

c. I used to do retrieval of client-supplied sps via lotus and the result is ok. 
lotus client retrieve --provider f03027241 bafykbzaceau5lzxz6afvfcxsu2ltu4iq2rnhxinpxubq5sphmjf3jxeiy4spu
lotus client retrieve --provider f03027241 bafykbzaceahuwek27ai7lc23hi2lmi6gj4aj5q2lv5tquytwdiglqwylbwnna

However, currently governance team commends that only spark's retrieval results are supported, which requires the client to inform sp to update the retrieval program. It may need some time. So I don't think the results of the review for sp retrieval are the final ones this time.

**Comment by cryptoAmandaL** [comment]
@Kevin-FF-USA Thanks Kevin, I've attended this wonderful meeting.

**Comment by galen-mcandrew** [comment]
Based on a deeper diligence review, it seems there is a glaring inconsistency here. Specifically, this allocator applied to build and run a market-based onboarding platform. Here is a quote from their [application](https://github.com/filecoin-project/notary-governance/issues/1027):

> We've innovated with a smart contract-based allocation process for transparency and efficiency, reducing bias and abuse risks.

However, that does not appear to be the case. This team is using the allocator.tech tooling, working currently with 3 clients and tracking diligence. This appears to be a manual diligence allocator.  

**Given this evidence, we will need to update the documentation to reflect the reality of this manual pathway.** If this team is in fact building a market-based platform that utilizes smart contracts for data distribution, we would love to see that! We encourage them to reapply for that pathway under the new rolling application structure.

In addition to updating this team to the correct pathway type, this allocator is demonstrating mixed compliance. Specifically:

- Mixed evidence of diligence with clients (no verification of client claims)
- Subsequent allocations given despite noncompliant client deal-making, with minimal allocator intervention through comments
- No retrievability for datasets, despite claims of public open data by both allocator and client (not showing distributed network data storage utility)

Given this mixed review, we are requesting that the allocator verify that they will uphold **all aspects & requirements of their initial application.** If so, we will request an **additional 2.5PiB of DataCap** from RKH, to allow this allocator to show increased diligence and alignment.

 **@cryptoAmandaL   can you verify that you will enforce program and allocator requirements?** (for example: public diligence, tranche schedules, and public scale retrievability like Spark). Please reply here with acknowledgement and any additional details for our review.

**Comment by cryptoAmandaL** [comment]
@galen-mcandrew 
Thanks for your comment. Our original goal was to use automated tools to form an innovative distribution pathway. However, due to difficulty of development and the actual situation, we decided to respond to clients' applications with priority, manually allocating datacap to them. We are following the rules as an allocator.

Regarding the retrieval, as https://github.com/filecoin-project/Allocator-Governance/issues/20#issuecomment-2147276129 mentioned, we used lotus to verify the client's sp, and the results are fine. In order to help our clients solve the problem of their sps not showing on spark, we have collected information on slack and github to provide to our clients, so that they can go and communicate with the sp. (https://filspark.com/troubleshooting-miner-score / https://app.slack.com/client/TEHTVS1L6/C015G3B5RQE)
![image](https://github.com/filecoin-project/Allocator-Governance/assets/134273610/d77aa0ff-8f93-49f9-b320-4a87acee1401)
Now I have opened an issue https://github.com/filecoin-station/spark/issues/75 to spark team. We have been looking for the solution.

In future work as allocator, we will continue to focus on the results of retrieval. Also help clients to get feedback and solve problems in retrieval and demand clients meet the standard. Further allocation of datacap to clients will be considered only after the client has responded to problem. We'll follow the response of spark team and updates of clients' sps, then make the results publicly on github.
